meta/main.yml

```diff
@@ -0,0 +1,63 @@
+galaxy_info:
+  author: New Relic
+  namespace: newrelic
+  description: Role for New Relic's Targeted Installs 
+  company: New Relic, Inc
+  role_name: install
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: license (GPL-2.0-or-later, MIT, etc)
+
+  min_ansible_version: "2.1"
+
+  # If this a Container Enabled role, provide the minimum Ansible Container version.
+  # min_ansible_container_version:
+
+  #
+  # Provide a list of supported platforms, and for each platform a list of versions.
+  # If you don't wish to enumerate all versions for a particular platform, use 'all'.
+  # To view available platforms and versions (or releases), visit:
+  # https://galaxy.ansible.com/api/v1/platforms/
+  #
+  # platforms:
+  # - name: Fedora
+  #   versions:
+  #   - all
+  #   - 25
+  # - name: SomePlatform
+  #   versions:
+  #   - all
+  #   - 1.0
+  #   - 7
+  #   - 99.99
+
+  galaxy_tags:
+    - monitoring
+    - observability
+    - scalability
+    - performance
+    - infrastructure
+    - cloud
+    - logging
+    - host
+    - linux
+    - windows
+    - .NET
+    - Java
+    - PHP
+    - NodeJS
+    - apm
```

Comment:
We should remove these galaxy tags for now

Reply:
True. Thanks @rthorn-nr!